src/quo2/components/password/tips/view.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.password.tips.view
+  (:require [react-native.core :as rn]
+            [quo2.components.password.tips.style :as style]
+            [quo2.components.markdown.text :as text]))
+
+(defn view
+  "Options
+   - `text` Password tip string
+   - `completed?` Password tip state"
+  [{:keys [completed? text]}]
+  [rn/view
+   {:style               style/container
+    :accessibility-label :password-tips}
+   [text/text
+    {:style  (style/tip-text completed?)
+     :weight :regular
+     :size   :paragraph-2} text]
+   (when completed? [rn/view {:style style/strike-through}])])
```

Comment:
we could add an accessibility label (or test id) to this view component so that it can be tested to check it renders 👍

Reply:
Which one should I use in this case accessibility label/test ID? Any preferences?

Reply:
hmm, actually an accessibility label is probably good to have here. 👍. You might need to add `:accessible true` prop as well for it to work